lib/Jsrt/Jsrt.cpp

```diff
@@ -2837,6 +2858,21 @@ CHAKRA_API JsCreateNamedFunction(_In_ JsValueRef name, _In_ JsNativeFunction nat
     });
 }
 
+CHAKRA_API JsCreateEnhancedFunction(_In_ JsEnhancedNativeFunction nativeFunction, _In_opt_ JsValueRef metadata, _In_opt_ void *callbackState, _Out_ JsValueRef *function)
+{
+    return JsCreateEnhancedFunctionHelper<false>(nativeFunction, metadata, callbackState, function);
+}
+
+CHAKRA_API JsCreateFunction(_In_ JsNativeFunction nativeFunction, _In_opt_ void *callbackState, _Out_ JsValueRef *function)
+{
+    return JsCreateNamedFunction(JS_INVALID_REFERENCE, nativeFunction, callbackState, function);
```

Comment:
`JsCreateNamedFunction` states that the `name` property is an `_In_` parameter, but `JS_INVALID_REFERENCE` is null so prefast is complaining here.

Reply:
Noticed and this is unfortunate. I'll change `JsCreateFunction` to call `JsCreateEnhancedFunctionHelper` directly since the `metadata` argument is marked as `_In_opt_`.